docs/src/_reference/command-line-interface.md

```diff
@@ -1341,53 +1341,6 @@ meltano test <plugin1>:<test-name1> <plugin2>:<test-name2>
 
 The `test` command can accept the `--environment` flag to target a specific [Meltano Environment](https://docs.meltano.com/concepts/environments). The [`default_environment` setting](https://docs.meltano.com/concepts/environments#default-environments) in your `meltano.yml` file will be applied if `--environment` is not provided explicitly.
 
-## `ui`
-
-- `meltano ui`: Start the Meltano UI.
-
```

Comment:
@tayloramurphy , what do you think of keeping this small reference, but with a deprecation note and a pointer to the troubleshooting section?

Reply:
I'm cool with that!